base_address_parish/models/res_parish.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022 Riverminds Cia Ltda - Mamfredy Mejia Matute
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, fields, models
+
+
+class ResParish(models.Model):
+    _name = "res.parish"
+    _order = "name"
+
+    name = fields.Char(required=True, translate=True)
+    zip = fields.Char()
+    city_id = fields.Many2one("res.city", "City", required=True)
+    state_id = fields.Many2one(
+        "res.country.state",
+        "State",
+        readonly=True,
+        domain="[('country_id', '=', country_id)]",
```

Comment:
`domain `is unnecesary if field is related and readonly=True

Reply:
OK